src/main/webapp/app/communication/answer-post/answer-post.component.scss

```
@@ -55,11 +55,89 @@
+    display: inline-flex;
+    align-items: center;
+    gap: 0.35rem;
+    padding: 0.3rem 0.65rem;
+    border: 1px solid var(--bs-border-color);
+    border-radius: 0.15rem;
+    background: #fff;
+
+    :root[prime-ng-use-dark-theme='true'] & {
+        background: var(--bs-body-bg);
+    }
+    cursor: pointer;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)